web/blog/public-roadmap-and-open-rfcs/index.md

```diff
@@ -0,0 +1,111 @@
+topics. This blog post explains why and how we did it.
+
+[roadmap]: https://vast.io/roadmap
+[rfc]: /docs/develop-vast/contributing/rfc
+
+<!--truncate-->
+
+As a community-first open-source project, we constantly strive for increasing
+transparency. Our long-term goal is establishing a fully open governance model.
+This will allow for a clear delineation of the open-source project and unbiased
+commercial offerings that we, the engineering team behind VAST at
+[Tenzir](https://tenzir.com), provide on top. Until we have bootstrapped
+ourselves and an active community, we aim for the right balance between open
+and closed.
+
+One step in the direction of open is publishing our [roadmap][roadmap] and
+enabling the community to participate in the planning through an [Request For
+Comments (RFC)][rfc] process.
+
+## Public Roadmap
+
+In the process of opening the roadmap, we had to answer several questions:
+
+1. **Audience**: should the content be for users only? What about developers?
+   Should we only mention features or also refactorings?
+
+2. **Interaction**: should this just be a read-only page or something the
+   community can directly interact with?
+
+3. **Tooling**: What is the right tool to encode the roadmap?
+
+Let's go through them one by one.
+
+Regarding audience, we want to avoid an overly narrow target group, as we are in
+phase of growth where breadth instead of depth is more important. Moreover, we
+gain more transparency if we can unveil all ongoing thrusts. Therefore, we want
+to cover the full spectrum of personas, but make it possible for each individual
+type of persona to get a relevant view.
+
+Regarding interaction, we are actively looking for engagement. Throwing a
+read-only version over the fence to the community is certainly informational,
+but we are looking for creating dialogue. Therefore, we want to allow everyone
+to discuss the various roadmap items in the open.
+
+Regarding tooling, we are in need for something that integrates well with the
+existing environment. Our GitHub presence includes code, documentation, website
+content, and third-party integrations. We also promote use of GitHub Discussions
+to engage with us. This makes GitHub the focal point to engage with the content.
+Therefore, we decided to encode the roadmap as GitHub issues; for clarity in a
+dedicated repository at <https://github.com/tenzir/public-roadmap>.
+
+We decided against dual-purposing the issue tracker of our main repository
+<https://github.com/tenzir/vast> because it would add roadmap items as many
+open, long-running issues that scatter the attention and potentially confuse the
+community. That said, the primary value of the issue tracker is the layer on top
+of issues: [Projects][github-projects], which allows for organizing issues
+across multiple dimensions in a visually appealing way. Here's how it looks as
+of today:
+
+[![Github Roadmap](github-roadmap.jpg)][roadmap]
```

Comment:
We need to recreate this screenshot after publishing the public parts, and then add a small section on the items not visible to the public.

Reply:
I've rebased and removed the screenshot, so that we can add it at the very end with:

```markdown
Here's how it looks as of today:

[![Github Roadmap](github-roadmap.jpg)][roadmap]
```